[
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1069320071",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/10352#issuecomment-1069320071",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/10352",
        "id": 1069320071,
        "node_id": "IC_kwDOCVq1mM4_vIeH",
        "user": {
            "login": "denizbeser",
            "id": 12912953,
            "node_id": "MDQ6VXNlcjEyOTEyOTUz",
            "avatar_url": "https://avatars.githubusercontent.com/u/12912953?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/denizbeser",
            "html_url": "https://github.com/denizbeser",
            "followers_url": "https://api.github.com/users/denizbeser/followers",
            "following_url": "https://api.github.com/users/denizbeser/following{/other_user}",
            "gists_url": "https://api.github.com/users/denizbeser/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/denizbeser/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/denizbeser/subscriptions",
            "organizations_url": "https://api.github.com/users/denizbeser/orgs",
            "repos_url": "https://api.github.com/users/denizbeser/repos",
            "events_url": "https://api.github.com/users/denizbeser/events{/privacy}",
            "received_events_url": "https://api.github.com/users/denizbeser/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2022-03-16T16:26:04Z",
        "updated_at": "2022-03-16T16:26:04Z",
        "author_association": "NONE",
        "body": "Having the same issue on python application with T5 onnx. The encoder onnx works, but the decoder onnx raises the same error:\r\n`\r\n2022-03-16 15:32:42.907044303 [E:onnxruntime:log, tensorrt_execution_provider.h:51 log] [2022-03-16 15:32:42   ERROR] 4: [graphShapeAnalyzer.cpp::processCheck::581] Error Code 4: Internal Error (IAssertionLayer (Unnamed Layer* 9) [Assertion]: condition[0] is false: 0. For input: 'input_ids' all named dimensions that share the same name must be equal. Note: Named dimensions were present on the following axes: 1 (name: 'sequence'), 1 (name: 'sequence'))`\r\n`Signal (11) received.`\r\n",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1069320071/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    }
]